scripts/new_project.kts

```diff
@@ -238,9 +262,22 @@ object NewProject {
         }
     }
 
-    private fun executeCommand(command: String) {
+    /**
+     * Execute a shell command
+     *
+     * Runtime.getRuntime().exec(String) will partition a command automatically, based on white spaces.
+     * If a file path contains a white space, it will not be able to find the file and result in an error.
+     * 
```

Comment:
I think adding this space is redundant 😄 

![Screen Shot 2565-08-16 at 11 33 27](https://user-images.githubusercontent.com/13492460/184798475-334ffe5f-9ef2-4f56-88f5-87d7e40ac57e.png)